tests/components/application-connector/README.md

```diff
@@ -229,4 +245,80 @@ To run the tests without removing all the created resources afterwards, run them
 
    ``` shell
    make clean-gateway-test enable-sidecar-after-mtls-test
+   ```
+
+## Application Connectivity Validator
+
+### Design and architecture
+
+The tests consist of:
+- [Test resources](./resources/charts/application-connectivity-validator-test/templates/) used to perform the test
+- [Test runner](./test/application-connectivity-validator/) with all the test cases
+
+The tests are executed as a Kubernetes Job on a Kyma cluster where the tested Application Connectivity Validator is installed. The test Job is deployed in the `test` Namespace.
+
+![Connectivity Validator tests architecture](./assets/connectivity-validator-tests-architecture.svg)
+
+> **NOTE:** Port `8080` must be excluded from redirection to Envoy, otherwise Connectivity Validator test Pod cannot pass the `X-Forwarded-Client-Cert` header to Connectivity Validator.
```

Comment:
```suggestion
> **NOTE:** Port `8080` must be excluded from redirection to Envoy, otherwise test Connectivity Validator test Pod cannot pass the `X-Forwarded-Client-Cert` header to Connectivity Validator.
```